src/mmw/js/src/core/chart.js

```diff
@@ -94,8 +95,13 @@ function renderHorizontalBarChart(chartEl, data, options) {
     }
 
     function updateChart() {
-        if($(svg).is(':visible')) {
+        var width = $svg.width(),
+            availableWidth = width - 240,
```

Comment:
It seems like the constant 240 should actually be a variable that changes as a function of the margins.